org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/IDebugSession.java

```diff
@@ -14,6 +14,7 @@
 import java.util.List;
 
 import com.sun.jdi.ThreadReference;
+import com.sun.jdi.request.EventRequestManager;
 
 public interface IDebugSession {
```

Comment:
removing is already handled. Each time debugger sends an exception breakpoint request to debuggee, it would first delete old requests